src/FsAutoComplete.Core/CodeGeneration.fs

```diff
@@ -19,7 +19,7 @@ type CodeGenerationService(checker : FSharpCompilerServiceChecker, state : State
       option {
         let! text = state.TryGetFileSource fileName |> Option.ofResult
         try
-            let! line = text.GetLine (Position.mkPos (i - 1) 0)
+            let! line = text.GetLine (Position.mkPos i 0)
```

Comment:
this was the core issue - we don't need to do a -1 line position because this helper covers that already.